domain/status/index.test.js

```diff
@@ -0,0 +1,113 @@
+    expect(statusCreated).toHaveProperty('companyId')
+  })
+})
+
+describe('update status', () => {
+  let statusCreated = null
+
+  beforeAll(async () => {
+    statusCreated = await StatusDomain.create({
+      ...fakerStatus(),
+      companyId: 'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    })
+  })
+  it('update status', async () => {
+    const statusMock = fakerStatus()
+
+    expect.hasAssertions()
+
+    const statusUpdated = await StatusDomain.update({
+      id: statusCreated.id,
+      ...statusMock,
+      companyId: 'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    })
+
+    expect(statusUpdated).toHaveProperty('id', statusCreated.id)
+    expect(statusUpdated).toHaveProperty('activated', statusMock.activated)
+    expect(statusUpdated).toHaveProperty('label', statusMock.label)
+    expect(statusUpdated).toHaveProperty('value', statusMock.value)
+    expect(statusUpdated).toHaveProperty('color', statusMock.color)
+    expect(statusUpdated).toHaveProperty('type', statusMock.type)
+    expect(statusUpdated).toHaveProperty('typeLabel', statusMock.typeLabel)
+    expect(statusUpdated).toHaveProperty(
+      'fakeTransaction',
+      statusMock.fakeTransaction
+    )
+    expect(statusCreated).toHaveProperty(
+      'companyId',
+      'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    )
+  })
+})
+
+describe('getById status', () => {
+  let statusCreated = null
+
+  beforeAll(async () => {
+    statusCreated = await StatusDomain.create({
+      ...fakerStatus(),
+      companyId: 'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    })
+  })
+  it('getById status', async () => {
+    expect.hasAssertions()
+    expect(true).toBeTruthy()
+
+    const getStatusById = await StatusDomain.getById(statusCreated.id)
+
+    expect(getStatusById).toHaveProperty('id', statusCreated.id)
+    expect(getStatusById).toHaveProperty('activated', statusCreated.activated)
+    expect(getStatusById).toHaveProperty('label', statusCreated.label)
+    expect(getStatusById).toHaveProperty('value', statusCreated.value)
+    expect(getStatusById).toHaveProperty('color', statusCreated.color)
+    expect(getStatusById).toHaveProperty('type', statusCreated.type)
+    expect(getStatusById).toHaveProperty('typeLabel', statusCreated.typeLabel)
+    expect(getStatusById).toHaveProperty(
+      'fakeTransaction',
+      statusCreated.fakeTransaction
+    )
+    expect(getStatusById).toHaveProperty(
+      'companyId',
+      'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    )
+  })
+})
+
+describe('getAll status', () => {
+  beforeAll(async () => {
+    await StatusDomain.create({
+      ...fakerStatus(),
+      companyId: 'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    })
+  })
+  it('getAll status', async () => {
+    expect.hasAssertions()
+    expect(true).toBeTruthy()
+
+    const getAllStatus = await StatusDomain.getAll(
+      {},
+      'co_4095e6c0-056d-4b6d-b857-a35584634ad0'
+    )
+
+    expect(getAllStatus).toHaveProperty('rows', getAllStatus.rows)
```

Comment:
segundo parametro é sempre verdade do jeito que está sendo feito
```suggestion
    expect(getAllStatus).toHaveProperty('rows')
```